src/Expression.cpp

```diff
@@ -48,7 +48,7 @@ ExpressionCommand &  ExpressionRoot::add_command (std::string name)
    ExpressionCommand * ptr = new ExpressionCommand;
    ptr->name = name;
 
-   expressions.push_back (std::move (std::unique_ptr <Expression> (ptr)));
+   expressions.emplace_back (std::unique_ptr <Expression> (ptr));
```

Comment:
Totally agree with that. But it would imply to either upgrade the c++ dialect to C++14 (required to use the `std::make_unique` function) or to provide an implementation of it.